File: 1. Two Sum/main.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        d={}
        l=len(nums)
        for i in range(l):
            if target-nums[i] not in d.keys():
                d[nums[i]]=i
            else:
                break
                
        l=[i,d[target-nums[i]]]
        #l.sort()

            
        
        return l

File: 1. Two Sum/main.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int l=nums.size(),i;
        map<int,int> m;
        
        for(i=0;i<l;i++){
            if(m.count(target-nums[i])==0)
                m[nums[i]]=i;
            else
                break;
        }
        return vector<int>({i,m[target-nums[i]]});
    }
};
